includes/Modules/Analytics.php

```diff
@@ -159,6 +166,25 @@ function( $data ) {
 		);
 	}
 
+	/**
+	 * Checks whether or not tracking snippet should be contextually disabled for this request.
+	 *
+	 * @return bool
```

Comment:
This needs a `@since` annotation.

Reply:
Do all methods need this or only public? (I thought the latter)

Reply:
We should have this on all methods.